src/analysis/souffle/sql_output.dl

```diff
@@ -0,0 +1,42 @@
+//-----------------------------------------------------------------------------
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//-----------------------------------------------------------------------------
+#ifndef SRC_ANALYSIS_SOUFFLE_SQL_OUTPUT_DL_
+#define SRC_ANALYSIS_SOUFFLE_SQL_OUTPUT_DL_
+
+#include "src/analysis/souffle/taint.dl"
```

Comment:
Also, shouldn't we reuse this file  https://github.com/google-research/raksha/blob/main/src/backends/policy_engine/souffle/sql_policy.dl ? 

If there is a strong reason to keep it here, let us consolidate these files in one location?

Reply:
I guess my thinking is that these `.dl` files and the rules used to generate their code, libraries, and binaries are a bit of a mess. They also can't really be well separated in a modular fashion: although we can organize different concepts into `.dl` files to help keep our own heads straight, they're still going to be rolled into one big ball of code by the C preprocessor, which will hand a single TU to the Souffle compiler. Because of this entangledness, I'd like to keep all the `.dl` files in the `//src/analysis/souffle` directory and expose only the libraries that should be linked with to other parts of the source tree.

Reply:
Renamed this to `policy_verifier_interface.dl` in recognition of the fact that we want this interface to be the generic Datalog policy verification interface, even if it has a few SQL-specific items in it at the moment.